package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -1156,6 +1156,16 @@ dnsmasq_start()
 	[ -n "$user_dhcpscript" ] && procd_set_param env USER_DHCPSCRIPT="$user_dhcpscript"
 	procd_set_param respawn
 
+	local instance_ifc instance_netdev
+	config_get instance_ifc "$cfg" interface
+	if [ -n "$instance_ifc" ]; then
+		network_get_device instance_netdev "$instance_ifc" && {
+			if [ -n "$instance_netdev" ]; then
```

Comment:
Why not use `&&` here just like in the line above (there maybe `if ...; then ... fi` would be better because it's more than one line).

Reply:
Okay, I'll replace `&&` with `if`.